drivers/media/i2c/ov5647.c

```diff
@@ -715,6 +565,13 @@ static int ov5647_set_mode(struct v4l2_subdev *sd)
 	if (ret < 0)
 		return ret;
 
+	ret = ov5647_write_array(sd, ov5647_common_regs,
```

Comment:
Do we want a guard like ``if (!ov5647->common_regs_written)`` like we do with other drivers?

Reply:
The `{0x0103, 0x01},` as the second register in the common list is a reset.
The mode tables do not all cover the superset of those set by any of the modes, hence requiring the reset.

Another followup is to remove all the registers that are set to the default value - there are a fair number of those which are all redundant.

Reply:
In which case, let's merge as-is.